metric_learn/base_metric.py

```diff
@@ -1,8 +1,43 @@
-from numpy.linalg import inv, cholesky
-from sklearn.base import BaseEstimator, TransformerMixin
+from numpy.linalg import cholesky
+from sklearn.base import BaseEstimator
 from sklearn.utils.validation import check_array
 
 
+class TransformerMixin(object):
```

Comment:
Does any class other than `BaseMetricLearner` use this mixin? If not, I'd just inline it into `BaseMetricLearner`.

Reply:
Done